docs/data/joy/components/avatar/avatar.md

```diff
@@ -28,58 +28,76 @@ export default function MyApp() {
+Wrap it around a string to display plain text, or use the `src` prop to display an image.
 
-### Variants
+## Anatomy
 
-The avatar component supports the four global variants: `soft` (default), `solid`, `outlined`, and `plain`.
+The Avatar component is composed of a root `<div>` that may wrap around an `<svg>`, an `<img>`, or a string:
 
-{{"demo": "AvatarVariants.js"}}
+```html
+<div class="JoyAvatar-root">
+  <!-- Avatar contents -->
+</div>
+```
 
-:::success
-To learn how to add more variants to the component, check out [Themed components—Extend variants](/joy-ui/customization/themed-components/#extend-variants).
-:::
+## Customization
 
-### Initials
+### Text Avatar
 
-Use a string as children to display its initials on the avatar component.
+Wrap the Avatar component around a string to display text.
+Note that the Avatar is designed to comfortably fit two letters at most—for instance, a user's initials:
 
 {{"demo": "InitialAvatars.js"}}
 
-### Image
+### Image Avatar
 
-Insert images in the avatar by using a path inside the `src` prop, similar to how you'd do in the HTML `<img>` element.
-Make sure to to write a meaningful description in the `alt` prop.
+Insert images into the Avatar by defining a path inside the `src` prop, just like you would with an HTML `<img>` element.
+Make sure to to write a meaningful description for the `alt` prop.
 
 {{"demo": "ImageAvatars.js"}}
 
 ### Image fallbacks
 
-If an error occurs while loading the avatar image, it will fallback to alternatives in the following order:
+If an error occurs while loading the Avatar's image, it will fall back to the following alternatives (in this order):
 
-1. The provided children string.
-2. The first letter of the `alt` text.
-3. A generic icon.
+1. The provided child string
+2. The first letter of the alt text
+3. The default generic icon
 
 {{"demo": "FallbackAvatars.js"}}
 
-### With a badge
+### Variants
+
+The Avatar component supports Joy UI's four [global variants](/joy-ui/main-features/global-variants/): `soft` (default), `solid`, `outlined`, and `plain`.
+
+{{"demo": "AvatarVariants.js"}}
+
+:::success
+To learn how to add more variants to the component, check out [Themed components—Extend variants](/joy-ui/customization/themed-components/#extend-variants).
+:::
+
+### Sizes
 
-Combine the avatar component with the [`Badge`](/joy-ui/react-badge/) to extend what you can communicate with it.
+The Avatar component comes in three sizes: `sm`, `md` (the default), and `lg`:
+
+{{"demo": "AvatarSizes.js"}}
+
+:::success
+To learn how to add custom sizes to the component, check out [Themed components—Extend sizes](/joy-ui/customization/themed-components/#extend-sizes).
+:::
+
+## Usage with the Badge
+
+Combine the Avatar component with the [Badge](/joy-ui/react-badge/) to visually communicate more complex information about a user's status:
 
 {{"demo": "BadgeAvatars.js"}}
 
-### Group
+## Usage with the Avatar Group
```

Comment:
I think that it would be simpler with:

```
## Avatar group
```

When I read "Usage with" I head with another component, but the Avatar group is part of the Avatar component.